ethclient/ethclient_test.go

```diff
@@ -590,10 +599,6 @@ func testCallContract(t *testing.T, client *rpc.Client) {
 	if _, err := ec.CallContract(context.Background(), msg, big.NewInt(1)); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	// PendingCallContract
-	if _, err := ec.PendingCallContract(context.Background(), msg); err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
```

Comment:
prefer to commenting out instead of deleting here

will also be great if we can add comments

Reply:
Done